apisix/admin/plugins.lua

```diff
@@ -52,24 +62,37 @@ function _M.get(name)
             type = true,
             scope = true,
         })
-
-        if arg["subsystem"] == "stream" then
-            return 200, stream_plugins
+        local all_plugins = http_plugins
+        for k, v in pairs(stream_plugins) do
+            all_plugins[k] = v
         end
+        return 200, all_plugins
```

Comment:
We don't need to change this, the api is going to be deprecated, so just leave it as it was

Reply:
done. The behaviour of all=true is now as it was previously, with an extra warn log telling that this will be deprecated.